pmultiqc/modules/quantms/quantms.py

```
@@ -1013,6 +1135,61 @@ def draw_search_engine(self):
+
+        enzyme_list = list()
+        for mzid_path in self.mzid_paths:
+            enzyme_list.append(list(next(mzid.MzIdentML(mzid_path).iterfind("Enzyme")).get("EnzymeName", None).keys())[0])
+        enzyme_list = list(set(enzyme_list))
+        enzyme = enzyme_list[0] if len(enzyme_list) == 1 else "Trypsin"
+        psm['missed_cleavages'] = psm.apply(lambda x: self.cal_MissCleavages(x['PeptideSequence'], enzyme), axis=1)
+
+        # Calculate the ID RT Score
+        if "retention_time" not in psm.columns:
+            # MGF
+            if self.mgf_paths:
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)